.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -2,151 +2,254 @@
+1. - [ ] Add team members crummel, LoganBussell, and msimons as reviewers to
+         the PR. Get acceptance.
+      - Note: Don't merge the PR.  This is just used to generate the pre-embargo-date
+        tarball for our partners.  This will be ported to the public repo
+        later and become the official release.
+1. - [ ] Download the artifacts from the centos71_Online_BaseTarball PR
+         validation build onto a machine with Bash.
+      - From the "Artifacts" page, navigate to "Tarball centos71_Online_BaseTarball"
+        and download the `base_tarball_*.tar.gz` and `tarball_*-smoke-test-prereqs.tar.gz`
+        files.
+      - The smoke-test prereqs are platform-independent but should be taken
+        from this non-portable build, *not* CentOS7 Offline Portable.
+      - Can use <https://github.com/dotnet/source-build/blob/release/3.1/scripts/fetch-azdo-artifact-tarball.sh>
+        as a library.
+        - E.g. download the tarballs with `download_tarball`, and fix them up
+          with `fix_azdo_tarball` if necessary.
+1. - [ ] Rename tarball and smoke-test prereqs to human-friendly names.
+         (Can use `rename_tarball_inner_dir`.)
       - Refer to previous release's email threads for naming pattern.
-1.  - [ ] [Internal] Upload tarballs and smoke-test prereqs to blob storage.
-1.  - [ ] [Internal] Send the tarball to partners. Include info about how certain we are that this will be the final Microsoft build.
-      - Never overwrite a tarball. At least change the blob storage virtual dir to represent a new build. This can help avoid timing issues and make it more obvious if stale links were accidentally re-sent rather than new ones.
-1.  - [ ] SYNC POINT: Wait for Microsoft build release.
-      - Wait for dotnet/announcements post. <kbd>Watch</kbd> the repo to get emails so you don't have to poll.
+1. - [ ] Upload tarballs and smoke-test prereqs to the `dotnetclimsrc` blob
+         storage under the `source-build` container in `red-hat/3.1/` under a
+         new folder named for the runtime and SDK version.  The access key for
+         this blob storage is in the `EngKeyVault` vault in the "Dotnet
+         Engineering Services" subscription.
+1. - [ ] Complete manual validation steps
+      - For 3.1, the build is poison-checked during the CI build.  If you get
+        an error related to "leak detection" or "CheckForPoison", get the source-build
+        team involved as these can be difficult to diagnose and fix.
+      - [ ] Verify the packs included in the SDK are from source-build-reference-packages,
+        not source-build.
+        - After smoke-tests have run, in `testing-smoke/builtCli`, run
+          `strings sdk/<version>/Sdks/Microsoft.NET.Sdk/tools/net472/Microsoft.CSharp.dll
+          | grep "built by"` and make sure this includes `built by: SOURCEBUILD`.
+      - [ ] Verify the packs included in the SDK have XML documentation comments
```

Comment:
This step doesn't seem actionable.  How will the person running this step know if there is an issue or not?

Is this something a source-build team member should sign-off on?

Reply:
Added threshold for an error.  I don't know that we need team signoff on this one but I can add it if you feel the threshold is insufficient.